doc/Rules.md

```diff
@@ -478,7 +478,7 @@ Some text here
 Some more text here
 ```
 
-Note: this rule will not be triggered if there are multiple consecutive blank
+Note: this rule will not be triggered if there are multiple consecutive blanks
 lines inside code blocks.
```

Comment:
Note that the text here originally read "blank lines", so changing to "blanks" makes this incorrect.